cdap-api/src/main/java/co/cask/cdap/api/workflow/WorkflowForkConfigurer.java

```diff
@@ -52,6 +54,13 @@
   WorkflowForkConfigurer<? extends WorkflowForkConfigurer<T>> fork();
 
   /**
+   * Adds a condition to the current branch of the fork.
+   * @param condition the {@link Predicate} to be evaluated at the condition node
+   * @return the configurer for the condition
+   */
+  WorkflowConditionConfigurer<? extends WorkflowForkConfigurer<T>> condition(Predicate<WorkflowContext> condition);
+
```

Comment:
same here. `fork()` returns the `WorkflowForkConfigurer`, which has both methods `fork` and `condition`